recipes/cogdl/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "cogdl" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/THUDM/cogdl/archive/{{ version }}.tar.gz
+  sha256: 4c35bcc20e9bd7e9928c6e5a02d2ca974d42ad47255ad7dafd70cbec5da079d6
+
+build:
+  skip: true  # [win]
```

Comment:
Can you go ahead and put this back to `noarch: python`? We don't need all the checks to pass to merge, and it *drastically* reduces the package build load.

Looks like the only windows problem is that some deps aren't available? In that case, it just won't be installable on Windows; and if they ever appear, this package will "just work".

Reply:
Done!  Thanks for taking a look @dopplershift